x-pack/plugins/uptime/server/rest_api/monitors/monitor_list.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { UMRestApiRouteFactory } from '../types';
+import { CONTEXT_DEFAULTS } from '../../../../../legacy/plugins/uptime/common/constants';
+import { API_URLS } from '../../../../../legacy/plugins/uptime/common/constants/rest_api';
+
+export const createMonitorListRoute: UMRestApiRouteFactory = libs => ({
+  method: 'GET',
+  path: API_URLS.MONITOR_LIST,
+  validate: {
+    query: schema.object({
+      dateRangeStart: schema.string(),
+      dateRangeEnd: schema.string(),
```

Comment:
i guess we can can move to "to", "from" pattern here.

Reply:
I'd prefer to not dedicate time to that specific task in this unwieldy patch, especially since there weren't many other things we wanted to correct in review. I have created https://github.com/elastic/uptime/issues/173 dedicated to fixing this everywhere in the solution.